client/src/pages/create/CreatePageForm2.tsx

```diff
@@ -40,6 +45,26 @@ export default function CreatePageForm2({
           <ChevronDown />
         </span>
       </div>
+      <div className="flex flex-col items-start">
+        <RadioButton
+          checked={private_ === false}
+          label="Public"
+          value="false"
```

Comment:
would like to research if this is a good pattern, i just want radio buttons that can be type boolean ugh